sources/views/contacts/listInContacts.js

```diff
@@ -3,14 +3,60 @@ import {contacts} from "models/contacts.js";
 
 export default class ListInContactsView extends JetView {
 	config() {
+		const _ = this.app.getService("locale")._;
 		return {
-			view: "list",
-			select: true,
-			template: "#Name#, e-mail: #Email#"
+			rows: [
+				{
+					view: "list",
+					localId: "contactsList",
+					select: true,
+					template: "#Name#, e-mail: #Email# <span class='webix_icon wxi-close'></span>",
+					onClick: {
+						"wxi-close": (e, id) => this.deleteItem(id)
+					}
+				},
+				{
+					view: "button",
+					value: _("Add"),
+					click: () => this.addItemIntoList()
+				}
+			]
 		};
 	}
 
-	init(view) {
-		view.parse(contacts);
+	init(view, url) {
+		this.list = this.$$("contactsList");
+		this.list.sync(contacts);
+		this.list.attachEvent("onAfterSelect", (id) => this.setIdIntoUrl(id));
+
+		const idFromUrl = url[0].params.id;
+		const lengthOfContacts = contacts.config.data.length;
```

Comment:
do not check data by view.config. if you load data by url, load or sync, config.data will be empty. 
if you want to check whether the component has data, you can use `contacts.count()`